docs/languages/csharp/delegates-events.md

```diff
@@ -2,4 +2,42 @@
 
 > **Note**
 > 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+> This topic will be covered under the following articles:
+
+<style type="text/css">
```

Comment:
I did build it locally, and it does work correctly. It will add this style, only for this page.